channels/us.m3u

```diff
@@ -223,11 +223,11 @@ https://58cc65c534c67.streamlock.net/alkarmatv.com/alkarmame2.smil/playlist.m3u8
 https://5aafcc5de91f1.streamlock.net/alkarmatv.com/alkarmaus.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="AllSports.us" tvg-name="All Sports" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Sports",All Sports
 https://2-fss-2.streamhoster.com/pl_120/amlst:203932-1476320/playlist.m3u8
-#EXTINF:-1 tvg-id="AllWeddings.us" tvg-name="All Weddings" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/d61oA0y.png" group-title="",All Weddings
+#EXTINF:-1 tvg-id="AllWeddings.us" tvg-name="All Weddings" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/d61oA0y.png" group-title="Family",All Weddings
```

Comment:
Family genre means entertainment for all ages. Are you sure on this one?

Reply:
Thank you.

Reply:
I would classify it as "Entertainment". We *rarely* film all the steps of a wedding.

Reply:
Okay

Reply:
I noticed I have three of those All Weddings, is that how many you guys have? I have seen that a lot of this. When I'm looking in Kodi. It will show so many channels of the same stream, but my list has more? I'm in sync with upstream master on GitHub in both branches. If this pull request is approved, should I rebase?

Reply:
Curently we have three but I'm about to remove Plex/Wurl version. It's not working

Reply:
Okay. In Kodi it only shows one. I have seen that with many other channels. It may be the way it's sorted. I have tried by name and number. VLC is the same way.